src/ParseUser.js

```diff
@@ -873,7 +878,10 @@ const DefaultController = {
 
     json.className = user.constructor.name === ParseUser.name ? '_User' : user.constructor.name;
     return Storage.setItemAsync(
-      path, JSON.stringify(json)
+      path,
+      (!CoreManager.get('ENCRYPTED_USER') && process.env.PARSE_BUILD !== 'browser')
+        ? JSON.stringify(json)
```

Comment:
Can you add a check for Crypto  being null aka not browser? A test would be awesome.

I can see an issue with enabling this feature when not on browser.

Reply:
After check some things I changed the conditional and also created a test to check if all works well.